### PR TITLE
added hotkeys

### PR DESCRIPTION
**Pull request by natekrall**

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -119,7 +119,7 @@
           </div>
         </center>
       </div>
-    </div>
+    </div>1
```
Comment (varvind) [inline views/index.ejs]: was this intentional?

File: views/index.ejs [hunk 2]
```diff
@@ -128,6 +128,34 @@
   <br>
   <br>
   
+  <script>
+  document.onkeydown = function(e) {
+    if (e.which == 49) {
+	  alert("Go to Home Page");
```
Comment (varvind) [inline views/index.ejs]: Maybe say "Going to Home Page" instead

File: views/index.ejs [hunk 3]
```diff
@@ -128,6 +128,34 @@
...
+	  
+	  var newUrl = "/";
+      window.location.replace(newUrl);
+	  document.location.href = newUrl;
+	  
+    } else if (e.which == 50){
+	  alert("Go to Search Page");
```
Comment (varvind) [inline views/index.ejs]: Same here

File: views/index.ejs [hunk 4]
```diff
@@ -128,6 +128,34 @@
...
+	  
+	  var newUrl = "/politician_search";
+      window.location.replace(newUrl);
+	  document.location.href = newUrl;
+	  
+	} else if (e.which == 51){
+	  alert("Go to Map Page");
```
Comment (varvind) [inline views/index.ejs]: Same here

File: views/index.ejs [hunk 5]
```diff
@@ -128,6 +128,34 @@
   <br>
   <br>
   
+  <script>
+  document.onkeydown = function(e) {
+    if (e.which == 49) {
+	  alert("Go to Home Page");
+	  
+	  var newUrl = "/";
+      window.location.replace(newUrl);
```
Comment (varvind) [inline views/index.ejs:143]: Tab this over

File: views/pollingLocations.ejs [hunk 6]
```diff
@@ -71,6 +71,33 @@
   <title>Election Tracker</title>
```
Comment (varvind) [inline views/pollingLocations.ejs:71]: Use comments from index.ejs for these becasuse they are the same

File: views/searchPoliticians.ejs [hunk 7]
```diff
@@ -0,0 +1,29 @@
+  <body>
```
Comment (varvind) [inline views/searchPoliticians.ejs:1]: Use previous comments

**Review by varvind — CHANGES_REQUESTED** [review]

File: views/index.ejs (hunk 2 above)
Comment (natekrall) [reply views/index.ejs]: I was planning on taking those out and making only one alert appear at the beginning explaining navigation. Additionally I am going to add the hotkeys in parenthesis next to the pages so for example Home (1)

File: views/index.ejs (hunk 3 above)
Comment (natekrall) [reply views/index.ejs]: See earlier comment

File: views/index.ejs (hunk 4 above)
Comment (natekrall) [reply views/index.ejs]: See earlier comment

File: views/index.ejs (hunk 5 above)
Comment (natekrall) [reply views/index.ejs:143]: fixed

**Review by varvind — APPROVED** [review]

**Pull request merged**